scripts/setup-vcsmx-env.sh

```diff
@@ -3,10 +3,10 @@
 # This script sets up VCS for FPGA-level RTL simulation on the Millennium
 # cluster at Berkeley
 
-firesim_root=$(dirname $0)/..
+firesim_root=$(pwd)/../
 cwd=$(pwd)
 
-source /ecad/tools/xilinx/SDx/2017.1.op/settings64.sh > /dev/null
+source /ecad/tools/xilinx/Vivado/2017.4/settings64.sh > /dev/null
```

Comment:
You didn't need SDx for this? Interesting.

Reply:
Yeah